lib/ble/local.toit

```diff
@@ -341,18 +341,38 @@ class LocalCharacteristic extends LocalReadWriteElement_ implements Attribute:
     descriptors_.remove local-descriptor
 
   /**
-  Sends a notification or an indication, based on the properties of the characteristic.
+  Sets the value of the characteristic.
+
+  This value is returned when a client reads the characteristic. The $handle-read-request
+    function takes precedence over this value.
+
+  In most cases $write is sufficient and easier to use. The main reason to use this function
+    is to set a value without sending out any notification.
+  */
+  set-value value/io.Data:
+    ble-set-value_ resource_ value
+
+  /**
+  Sets the value of this characteristic and sends a notification or indication if supported.
 
   If the characteristic supports both indications and notifications, then a notification is sent.
+
+  If $update-value is true, sets the value of the characteristic to $value. Any read requests
```

Comment:
Should `--update-value` be `--set-value` or `set-value` be `update-value`?

Reply:
switched to `--set-value`.
done.